browser/ui/webui/brave_webui_source.cc

```diff
@@ -88,6 +88,7 @@ void CustomizeWebUIHTMLSource(const std::string &name, content::WebUIDataSource*
       }
     }, {
       std::string("rewards"), {
+        { "878e89ddb547d70019456c4754522212.ico", IDR_BRAVE_REWARDS_FAVICON },
```

Comment:
are these hashes or something? Where does this come from?

Reply:
Ok, so I generated this one manually on the fly and then meant to swoop back and use a hash generator but forgot. I noticed we use hashes and file names sorta interchangeably but I was just trying to follow the convention here.

Reply:
AFAIK these don't need to be hashes, it was just a result of some webpack config that's IMO not really suitable for having to include file paths as constants like this. In this instance it definitely doesn't need to be like this since it's not referenced by webpack, so I think we can just use the filename